Prevents OSError: dlopen failed: library "libc.so.6" not found
(at runtime)

--- a/torch/distributed/elastic/multiprocessing/redirects.py
+++ b/torch/distributed/elastic/multiprocessing/redirects.py
@@ -47,7 +47,7 @@
             "msvcr110, msvcr100). Redirects cannot function without a CRT."
         )
     else:
-        return ctypes.CDLL("libc.so.6")
+        return ctypes.CDLL("libc.so")
 
 
 libc = get_libc()
